src/sentry/issues/attributes.py

```diff
@@ -192,6 +204,8 @@ def _bulk_retrieve_snapshot_values(
             "group_id": group_value.id,
             "status": group_value.status,
             "substatus": group_value.substatus,
+            "priority": group_value.priority,
+            "first_release": group_value.first_release.id if group_value.first_release else None,
```

Comment:
I'd recommend just using `group_value.first_release_id`, and just storing the `first_release_id` in the `GroupValues` structure too